src/api_client_example/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ set(CMAKE_CXX_STANDARD 20)
 
 find_package(antlr4-runtime REQUIRED)
 find_package(yaml-cpp REQUIRED)
+find_package(fmt REQUIRED)
```

Comment:
Why does this client project needs to find_package ? Doesn't it "inherit" from antares's dependencies ?